### DISCUSSION TITLE
add a "player_start" item and a "end" trigger pls

### DISCUSSION THREAD

**Discussion by ARTUTTU** [BEEmod/BEE2-items#4217] [Feature Suggestions]
pls,is it posible?

**Comment by ARTUTTU** [comment]
please

**Comment by ARTUTTU** [comment]
i know that in hamer is called "info_player_start"

**Comment by FrozenDragon0** [comment]
This is what you should be using hammer for, the puzzlemaker is not designed for story maps.

**Reply by ARTUTTU** [reply]
yeah

**Reply by ARTUTTU** [reply]
I'm sorry for bothering
